demo/api/.eslintrc.json

```diff
@@ -1,4 +1,18 @@
 {
     "extends": "@comet/eslint-config/nestjs",
-    "ignorePatterns": ["src/db/migrations/**", "dist/**"]
+    "ignorePatterns": ["src/db/migrations/**", "dist/**"],
+    "rules": {
+        "no-restricted-imports": [
```

Comment:
should we also restrict it in @comet/eslint-config to bring it to the projects?

Reply:
we decided against this, as it is difficult to update existing applications

Reply:
I can't remember this.

How do we intend to handle new Projects then?

In my opinion we should do one of the following to have this rule in NEW projects:

- **Version 1:** add no-restricted-imports to @comet/eslint-config/nestjs.js
     -  deactivate in existing projects (manually or with codemod)
- **Version 2:** create a new version of  eslint-config/nestjs.js (e.g. nestjsv2.js)

I would tend to Version 1.

Reply:
> How do we intend to handle new Projects then?

New projects will be created with the Starter, so they will have the import restriction

> deactivate in existing projects (manually or with codemod)

The problem is that we can't disable no-restricted-import for a single import. Therefore disabling the rule would disable all import restrictions.

Reply:
it's not easy and maybe not straight forward, but it could work like this PoC targetting on the next branch: https://github.com/vivid-planet/comet/pull/3577